### PR TITLE
Added info about where to add/drop capabilities

### PR DESCRIPTION
**Pull request by vonneudeck**
The user-guide did not mention where to drop/add capabilities in Kubernetes.
Added a link to the not yet complete API documentation and an example.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1804)

### CODE REVIEW

File: docs/user-guide/containers.md [hunk 1]
```diff
@@ -94,3 +94,23 @@ The relationship between Docker's capabilities and [Linux capabilities](http://m
 | SETFCAP |  CAP_SETFCAP |
 | WAKE_ALARM |  CAP_WAKE_ALARM |
 | BLOCK_SUSPEND |  CAP_BLOCK_SUSPEND |
+
+Capabilities are added/dropped in the [`SecurityContext`](http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_securitycontext), e.g.:
```
Comment (devin-donnelly) [inline docs/user-guide/containers.md]: Passive voice. If the user is responsible, please rephrase to "You can add or drop capabilities by editing the [`SecurityContext`]."

**Comment by devin-donnelly** [comment]
@vonneudeck Thanks for the fix; this still needs a tech LGTM before I can merge.

**Comment by devin-donnelly** [comment]
@mikedanese , can you or someone else do a tech review on this?

**Comment by mikedanese** [comment]
@devin-donnelly I can do this. Sorry, fell under my radar.

**Comment by mikedanese** [comment]
Yaml looks correct. This is indeed how you drop capabilities. Tech LGTM.

**Pull request merged**